### DISCUSSION TITLE
Mysterium or subtle bug?

### DISCUSSION THREAD

**Discussion by klassikmann** [rern/rAudio#854] [Fixed Issues]
Is anyone able to explain this effect not related to the new release (20220317)? 

Look at this screenshot from the library view:
<img src="https://user-images.githubusercontent.com/25383513/158989412-82c66e51-56fa-4b14-aa88-02b458799140.jpg" width=600>
Why does "Jan Willem ..." appear after "Jordi Savall ..."?
If I go to the album view - sorted by artist - the nine symphonies conducted by "Jan Willem ..." appear last at letter "J". 

I couldn't detect any special character neither in the tags nor in the directory name:
<img src="https://user-images.githubusercontent.com/25383513/158994177-2fddcd78-c549-4e98-b2d4-37c4af6629a8.jpg" width=600>
If I sort manually the corresponding files in data/mpd everything is as expected ("Jan Willem ..." before "Jordi Savall ...").

Any clue?

Another (same?) effect:
<img src="https://user-images.githubusercontent.com/25383513/158994631-8addc0b9-d13f-4970-a6ba-58c852724594.jpg" width=600>
Why does "The Alan Parsons Project" appear after "The Allman Brothers Band"?

**Comment by EarsofTin** [comment]
I observe the same with respect to the Allmans and Alan Parsons.  A few months ago I posted that Stan Getz followed Stevie Wonder in my collection.  It's not a big deal but it makes me curious.  Perhaps the string "an" gets ignored in the sorting as a common article.

**Reply by klassikmann** [reply]
@EarsofTin 
> Perhaps the string "an" gets ignored in the sorting as a common article.

That's the case apparently. I changed "Jan Willem ..." to "Jon Willem ..." and the sorting in both the library and the album view is correct now.
=> Not a mysterium, it's a bug.

**Comment by klassikmann** [comment]
Category changed from "Testing" to "Bug Reports".

**Comment by Sagnasty** [comment]
Interesting find @klassikmann

Below, a screenshot from my library

![image](https://user-images.githubusercontent.com/76575081/159125161-ec697d84-3399-44ef-a345-c681e09a0e0e.png)

Jim

3 x Raspberry Pi 3B+ Rev 1.3 > HiFiBerry DAC+ Pro
1 x Raspberry Pi 3B Rev 1.2 > HiFiBerry DAC+ Pro
Synology DS718+

**Reply by klassikmann** [reply]
@Sagnasty 
> Interesting find

The honor goes to @EarsofTin 
He made the correct explanation.

**Comment by klassikmann** [comment]
@rern 
> If I go to the album view - sorted by artist - the nine symphonies conducted by "Jan Willem ..." appear last at letter "J".

Your change in cmd-listsort.php in the UPDATE branch fixes the problem in the library view but not in the album view automatically.
I have to perform an extra update database + thumbnails.

**Comment by klassikmann** [comment]
Fixed in rAudio 1 • 20220327
